widget/hyperlink.go

```diff
@@ -0,0 +1,140 @@
+package widget
+
+import (
+	"image/color"
+	"net/url"
+	"os/exec"
+	"runtime"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/theme"
+)
+
+// Hyperlink widget is a text component with appropriate padding and layout.
+// When clicked, the default web browser should open with a URL
+type Hyperlink struct {
+	textProvider
+	Text      string
+	Url       string
+	Alignment fyne.TextAlign // The alignment of the Text
+	TextStyle fyne.TextStyle // The style of the hyperlink text
+}
+
+// NewHyperlink creates a new layout widget with the set text content
+func NewHyperlink(text string, sUrl string) *Hyperlink {
```

Comment:
I think the prefix variable name convention does not fit in Go - just "url" should be used no?

Reply:
There's a package called "url" so that's why I was distinguishing.  I just tested and apparently go is smart enough to distinguish the two, so I'll change it :)

Reply:
Apologies I forgot about that. When that happens, and if usage is obvious, I've gone for a single letter, but I'm not confident that's better :( I'm OK with different names, but using the type prefix notation just seems out of place

Reply:
Changed in 3fdc033528934b7e8ed483474c570a7f82cbcc9a